modules/installation-azure-stack-hub-config-yaml.adoc

```diff
@@ -136,26 +155,26 @@ endif::openshift-origin[]
   - 172.30.0.0/16
 platform:
   azure:
-    armEndpoint: azurestack_arm_endpoint <1> <5>
-    baseDomainResourceGroupName: resource_group <1> <6>
-    region: azure_stack_local_region <1> <7>
-    resourceGroupName: existing_resource_group <8>
+    armEndpoint: azurestack_arm_endpoint <1> <6>
+    baseDomainResourceGroupName: resource_group <1> <7>
+    region: azure_stack_local_region <1> <8>
+    resourceGroupName: existing_resource_group <9>
     outboundType: Loadbalancer
     cloudName: AzureStackCloud <1>
-    clusterOSimage: https://vhdsa.blob.example.example.com/vhd/rhcos-410.84.202112040202-0-azurestack.x86_64.vhd <1> <9>
-pullSecret: '{"auths": ...}' <1> <10>
+    clusterOSimage: https://vhdsa.blob.example.example.com/vhd/rhcos-410.84.202112040202-0-azurestack.x86_64.vhd <1> <10>
```

Comment:
On the preview, you're missing <10> here. It goes <9>, <1>, <1> and <11>.

Reply:
I missed it at first as well. This particular line is long. If you scroll the sample the right you will see `<1>` and `<10>`